packages/@best/cli/package.json

```diff
@@ -34,7 +34,7 @@
     "rimraf": "~2.6.2",
     "simple-git": "~1.113.0",
     "simple-statistics": "^6.0.1",
-    "yargs": "~11.0.0"
+    "yargs": "~13.3.0"
```

Comment:
yargs@11.0.0 has 2 vulnerable dependencies: yargs-parser and os-locale > mem. Bumping yargs to 13 bumps yargs-parser to patched version and completely removes os-locale dependency